.test-conda-env-py3.yml

```diff
@@ -12,5 +12,15 @@ dependencies:
 - pyopencl
 - islpy
 - pyfmmlib
+
 # Only needed to make pylint succeed
 - matplotlib
+
+- pip
+- pip:
+    - git+https://github.com/inducer/pymbolic
+    - git+https://github.com/inducer/loopy
```

Comment:
FYI @mattwala It seems that there's not really a need for separate conda-specific requirements files.